pkg/project/expander.go

```diff
@@ -134,8 +137,27 @@ func ScriptExpander(_ string, name string, meta string, isFunction bool, project
 		return "", nil
 	}
 
-	if meta == "path" && isFunction {
-		return expandPath(name, script)
+	if isFunction {
+		switch meta {
+		case "path":
+			return expandPath(name, script)
+		case "slash-path":
+			path, err := expandPath(name, script)
+			if err != nil {
+				return "", err
+			}
+			return filepath.ToSlash(path), nil
+		case "bash-path":
+			path, err := expandPath(name, script)
+			if err != nil {
+				return "", err
+			}
+			if runtime.GOOS == "windows" {
+				return fmt.Sprintf("$((type cygpath > /dev/null && cygpath '%s') || (type wslpath > /dev/null && wslpath -u '%s') || echo '%s')", path, path, path), nil
```

Comment:
We need to run this ourselves in the background and pass back the result. `path()` returns a string that is the path of the file, not a string that needs to be evaled.